api/office/schema.py

```diff
@@ -113,17 +119,18 @@ def resolve_all_offices(self, info, **kwargs):
 
     def resolve_get_office_by_name(self, info, name):
         query = Office.get_query(info)
-        check_office = query.filter(OfficeModel.name == name).first()
+        resultActive = query.filter(OfficeModel.status == "active")
+        check_office = resultActive.filter(OfficeModel.name == name).first()
         if not check_office:
             raise GraphQLError("Office Not found")
 
         if name == "Epic tower":
-            exact_query = lagos_office_join_location(query)
+            exact_query = lagos_office_join_location(resultActive)
             result = exact_query.filter(OfficeModel.name == name)
             return result.all()
 
         else:
```

Comment:
You can remove this unnecessary blank line here.

Reply:
@kampkelly , which line specifically?

Reply:
Line 131 @michael-basweti

Reply:
@kampkelly there is no blank line at 131